src/Workers/WebARKitCVWorkers.ts

```diff
@@ -1,7 +1,7 @@
 import Worker from "worker-loader?inline=no-fallback!./Worker";
 import { ITrackable } from "../interfaces/Trackables";
 
-export abstract class AbstractWebARKitCVWorker{
+export abstract class AbstractWebARKitCVWorker {
     protected trackables: Map<number, ITrackable>;
     protected opencv: any;
```

Comment:
i think that opencv propertiy is not necessary, i can import it directly in the worker without problems.

Reply:
done with https://github.com/webarkit/webarkit-cv/pull/5/commits/e1406de98bfcb1a749518a77fd83f2c35acc56d8